portal-web/test/functional/com/liferay/portalweb/macros/WorkflowMetrics.macro

```diff
@@ -485,4 +513,27 @@ definition {
 		AssertElementPresent(locator1 = "WorkflowMetrics#METRICS_WORKLOAD_BY_ASSIGNEE_PAGE_TASKCOUNT");
 	}
 
+	macro viewWorkloadByAssigneeSingleApprover {
```

Comment:
Hi @calvinctvo ,
1) Would you please change the name of this macro from viewWorkloadbyAssigneeSingleApprover to only viewWorkloadByAssignee? 

2) I noticed that you are checking here if the elements are present. The idea of the macro is correct. Though, we should only see if METRICS_WORKLOAD_BY_ASSIGNEE_NAME_TABLE is present but for :

METRICS_WORKLOAD_BY_ASSIGNEE_TASKCOUNT_OVERDUE
METRICS_WORKLOAD_BY_ASSIGNEE_TASKCOUNT_ON_TIME
METRICS_WORKLOAD_BY_ASSIGNEE_TASKCOUNT_TOTAL_PENDING

We must validate the value and not if they are present or not. For example, we 1 instance attached to each user, and they are in Review task:

1 for 1user user1
1 for 2user user2
1 for 3user user3

When we go to Workload by Assignee, we can see the 3 instances

On Time - 0
Overdue - 0
Total - 0

When we filter for 1Role, the usage of the macro should be something like:

WorkflowMetrics.viewWorkloadByAssignee(
			assigneeName = "1user user1",
			isNotPresent = "2user user2,3user user3",
                         onTimeValue = "0",
                         overdueValue = "0",
                         totalValue = "1");

Inside the macro, we would have to assert that the numbers from "1user user1" are indeed correct.

isNotPresent variable is used only to check if METRICS_WORKLOAD_BY_ASSIGNEE_NAME_TABLE is present in the table.

Also, the way the macro is written, we can validate only 1 entry per call. If we want to validate if all entries are displayed in the table, we should call the macro N times:

WorkflowMetrics.viewWorkloadByAssignee(
			assigneeName = "1user user1",
			isNotPresent = "",
                         onTimeValue = "0",
                         overdueValue = "0",
                         totalValue = "1");

WorkflowMetrics.viewWorkloadByAssignee(
			assigneeName = "2user user2",
			isNotPresent = "",
                         onTimeValue = "0",
                         overdueValue = "0",
                         totalValue = "1");

WorkflowMetrics.viewWorkloadByAssignee(
			assigneeName = "3user user3",
			isNotPresent = "",
                         onTimeValue = "0",
                         overdueValue = "0",
                         totalValue = "1");

Please let me know if any questions about this comment.